ivre/db/sql/sqlite.py

```diff
@@ -90,3 +90,32 @@ def _insert_or_update(self, timestamp, values, lastseen=None):
                 self.tables.passive
             ).where(whereclause).values(upsert)
             self.db.execute(updt)
+
+    def update_dns_blacklist(self):
+        """ Update the current database to detect blacklist domains.
```

Comment:
I had missed this one, but since I have some other fixes to implement, can you remove the space before `U`?

Same for the generic implementation of this method.